diffsims/crystallography/reciprocal_lattice_vector.py

```diff
@@ -502,6 +503,11 @@ def scattering_parameter(self):
 
         return 0.5 * self.gspacing
 
+    def rotate_from_matrix(self, rotation_matrix):
```

Comment:
Where is this used or going to be used? If internally, I'd suggest to remove it and do the computation where necessary.

How is it going to be used? There is no docstring. The requirement is a (3, 3) matrix, this should be stated.

Reply:
@hakonanes This kind of has to be included. Otherwise, there isn't really a reason to use the RLV.  We need to rotate the vectors and then determine the intersection with the Ewald sphere. 

I'm open to suggestions...

We need a method that basically goes from RLV --> Rotate --> Select --> Create DiffractingVector

Reply:
There is a bit of a problem where once you rotate the diffraction vector the hkl no longer make sense. 

I'm not sure the proper solution though likely we would need to rethink the Miller/ReciporicalLatticeVector class.

Reply:
Now that I think about it, it might be fine that reciprocal lattice vectors cannot be rotated... If that functionality is needed, one can just use regular Miller (hkl) vectors instead. We can continue discussing in #211.